vms/authentication/templates/authentication/login.html

```diff
@@ -62,6 +62,11 @@
                 </div>
             </div>
         {% endif %}
+        {% if form.errors %}
+        <div class="alert alert-danger">
+            <i class="fa fa-warning"></i> "Your username and password didn't match. Please try again."
```

Comment:
@amruthasangeeth Are we using fa fa icons anywhere in VMS? If not, it is better to have it without this.